index.js

```diff
@@ -7,37 +7,55 @@ var INCLUDE_RE = /\!{3}\s*include\s*\(\s*(.+?)\s*\)\s*\!{3}/i;
 
 module.exports = function include_plugin(md, options) {
   var root = '.',
-      includeRe = INCLUDE_RE;
+      includeRe = INCLUDE_RE,
+      throwError = true,
+      notFoundMessage = 'File \'{{FILE}}\' not found',
+      circularMessage = 'Circular reference between \'{{FILE}}\' and \'{{PARENT}}\'';
 
   if (options) {
     if (typeof options === 'string') {
       root = options;
     } else {
       root = options.root || root;
       includeRe = options.includeRe || includeRe;
+      throwError = options.throwError || throwError;
+      notFoundMessage = options.notFoundMessage || notFoundMessage;
+      circularMessage = options.circularMessage || circularMessage;
     }
   }
 
   function _replaceIncludeByContent(src, rootdir, parentFilePath, filesProcessed) {
     filesProcessed = filesProcessed ? filesProcessed.slice() : []; // making a copy
-    var cap, filePath, mdSrc, indexOfCircularRef;
+    var cap, filePath, mdSrc;
 
     // store parent file path to check circular references
     if (parentFilePath) {
       filesProcessed.push(parentFilePath);
     }
     while ((cap = includeRe.exec(src))) {
       filePath = path.resolve(rootdir, cap[1].trim());
+      mdSrc = '';
```

Comment:
I would have used a new variable to handle the error message. Like `errorMessage`.